test/e2e/e2e_test.go

```diff
@@ -0,0 +1,225 @@
+package e2e
+
+import (
+	"os"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("For a Kubernetes cluster with Tekton installed", func() {
```

Comment:
I would prefer having a single data-structure documenting the tests cases, by expanding it as gink-go code it creates a number of repeated blocks, like for instance [here](https://github.com/redhat-developer/build/blob/4b5ea7bc2a47db846987de20b2a0a6fef1c0e2bb/test/e2e/e2e_test.go#L30-L33).

Reply:
There is a [table extension for Ginkgo](http://onsi.github.io/ginkgo/#table-driven-tests). But not sure if that works with parallelism and focus/skip. I do not want to change the code without further opinions and would never give up parallelism or focus/skip for it.

One thing to keep in mind: we currently have only good-case test specs. Once we start to test the exceptional cases as well - like for PR #158 I need to do this - the spec implementation will look different.

Reply:
Okay. I was referring to have a single `map` like before, and on iterating that `map` expand into `ginkgo` statements. However, having different test cases to implement does justify the current construction :-)